test/upgrade/security.test.ts

```diff
@@ -1,51 +1,51 @@
-import { expect } from 'chai'
-import hre, { upgrades } from 'hardhat'
-import '@nomiclabs/hardhat-ethers'
+// import { expect } from 'chai'
+// import hre, { upgrades } from 'hardhat'
```

Comment:
Why did this file get commented out?

Reply:
The test helped with testing the first Staking contract upgrade we did. I'm planning to do something different that will help with "generically" test any upgrade. Thinking about creating a docker image running hardhat node, deploy the last deployed version of contracts there, then checkout the new version and upgrade to see if everything works.